src/katello/repos.py

```diff
@@ -36,7 +37,10 @@ def report_enabled_repos(consumer_id, report):
         error_message(str(error))
 
 
-def upload_enabled_repos_report(report):
+def upload_enabled_repos_report(report, force=False):
```

Comment:
I know it's changing more places but it'd be nice if we didn't give any knowledge of  plugins being enabled or disabled to this method. ie I think the plugins and the executables themselves should handle it. What do you think?

If this gets merge after #77 then the DNF stuff won't have to be touched which is nice!

Reply:
I had put it here because its centralizing the check.  If you think about the config file as a  yum plugin config file i can see your point, if you think about it as a host-tools config file, maybe this makes more sense?    I don't have a strong opinion either way, just had put it here to keep from duplicating logic.

Reply:
I don't feel strongly about it either. Let's leave it this way and and change it when (if ever) we need to!